GCRCatalogs/helpers/base_filters.py

```diff
@@ -0,0 +1,69 @@
+"""
+Base filters. To be called by high-level helper functions that are specific
+to catalog types (e.g., tract catalogs).
+"""
+
+import numpy as np
+
+from GCR import GCRQuery
+
+__all__ = ["partition_filter", "sample_filter"]
+
+
+def partition_filter(name, ids, id_high=None):
```

Comment:
Would like to see either a more descriptive first argument name (`column_name`? `partition_name`?) or something in the docstring about it, e.g
``` Returns a GCRQuery object to be used in the `native_filters` argument of get_quantities with native_filter *name*```

Reply:
`partition_name` is better. Changed.